src/alire/alire-hashes-compute.adb

```diff
@@ -0,0 +1,41 @@
+with Alire.Errors;
+with Alire.OS_Lib.Subprocess;
+
+package body Alire.Hashes.Compute is
+
+   function Hash_File (Kind : Kinds;
+                       Path : Platform_Independent_Path) return Any_Hash
+   is
+      --  Determine the coreutils tool to use for hashing
+      Hasher : constant String := Utils.To_Lower_Case (Kind'Img) & "sum";
+      --  E.g.; sha512sum
+
+      Output    : Utils.String_Vector;
+      Exit_Code : Integer;
+   begin
+      OS_Lib.Subprocess.Raw_Spawn
+        (Program    => "sh",
```

Comment:
It would be better to use the hash functions provided in libGNAT. There would be less dependency on external tools.

Reply:
I thought we wanted this for simplicity. Will revert the change.

Reply:
Fixed in latest push.